api/processor_api.py

```diff
@@ -274,6 +274,50 @@ def handle_config_response(config, success):
                 )
             return JSONResponse(content=humps.decamelize(config))
 
+        def get_areas():
+            config = extract_config()
+            areas_name = [x for x in config.keys() if x.startswith("Area")]
```

Comment:
Why not use something like `config.get_sections()`. I think that makes the code clearer

Reply:
extract_config() already takes the code from there. I managed to reduce complexity in the gets adding them to the method, but on all other methods that need to edit the config I still need to retrieve and filter separately through the whole config